libs/authorizenet/src/services/accept-js-production.token.ts

```diff
@@ -0,0 +1,12 @@
+import { InjectionToken } from '@angular/core';
+
+/**
+ * A token that represents the version to use for Accept.JS calls
+ * By default, we assume you're using the sandbox (false) for safety purposes.
```

Comment:
I think you should state what the true value represents, even though it's pretty obvious.

Reply:
Yep, explained.